chain/indexer/src/streamer/mod.rs

```diff
@@ -382,6 +392,7 @@ pub async fn start(
                 Box::pin(build_streamer_message(&view_client, block, &shard_tracker)).await;
             let Ok(streamer_message) = streamer_message else {
                 tracing::error!(target: INDEXER, ?block_height, ?streamer_message, "Failed to build StreamerMessage. Skipping.");
+                streamer_message.unwrap();
```

Comment:
Should this be removed? This change feels accidental. If we want to panic in this case, it's better to do so explicitly.

Reply:
yes, this is unintended, sorry 🤦